deployment/helm/templates/udev-configuration.yaml

```diff
@@ -8,7 +8,7 @@ spec:
     name: udev
     discoveryDetails: |+
       udevRules:
-      {{- required "Please set at least one udev rule with `--set udev.configuration.discoveryDetails.udevRules[0]==\"<udev rule>\"' to specify what you want discovered. See the udev Configuration document at https://github.com/deislabs/akri/blob/main/docs/udev-configuration.md for more information." .Values.udev.configuration.discoveryDetails.udevRules | toYaml | nindent 6 }}
+      {{- required "Please set at least one udev rule with `--set udev.configuration.discoveryDetails.udevRules[0]==\"<udev rule>\"' to specify what you want discovered. See the udev Configuration document at https://docs.akri.sh/discovery-handlers/udev for more information." .Values.udev.configuration.discoveryDetails.udevRules | toYaml | nindent 6 }}
```

Comment:
This change to the template is what is kicking off the end to end tests. I think it makes sense to still not do a version update. This log will be updated in a later chart.